### PR TITLE
image merge

### PR DESCRIPTION
**Pull request by Omid07**

### CODE REVIEW

File: app/src/main/java/com/example/omid/ps01/MainActivity.java [hunk 1]
```diff
@@ -68,5 +82,50 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
...
+                        Uri uri = item.getUri();
+                        Cursor cursor = getContentResolver().query(uri, filePathColumn, null, null,
+                                null);
+                        cursor.moveToFirst();
+                        int columnIndex = cursor.getColumnIndex(filePathColumn[0]);
+                        String picturePaths = cursor.getString(columnIndex);
+                        paths.add(picturePaths);
+                        cursor.close();
+                    }
+                    if (paths.size() == 2) {
```
Comment (b1acKr0se) [inline app/src/main/java/com/example/omid/ps01/MainActivity.java]: define a constant for 2

File: app/src/main/java/com/example/omid/ps01/MainActivity.java [hunk 2]
```diff
@@ -68,5 +82,50 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
...
+                }
+            }
+        }
+    }
+
+    @Override
+    public void onClick(View v) {
+        switch (v.getId()) {
+            case R.id.button_camera:
+                camera();
```
Comment (b1acKr0se) [inline app/src/main/java/com/example/omid/ps01/MainActivity.java]: please use a verb as prefix for method name, i.e `openCamera()`

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -22,4 +22,11 @@
         android:layout_height="wrap_content"
         android:layout_toRightOf="@+id/button_camera"
         android:text="@string/gallery"/>
+
+    <Button
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:id="@+id/button_merge"
```
Comment (b1acKr0se) [inline app/src/main/res/layout/activity_main.xml]: please format this file

File: app/src/main/res/layout/activity_merge_image.xml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fillViewport="true">
+
+    <RelativeLayout
```
Comment (b1acKr0se) [inline app/src/main/res/layout/activity_merge_image.xml]: `LinearLayout` is made for this

**Comment by b1acKr0se** [comment]
OK

**Comment by nvmanh** [comment]
OK!

**Pull request merged**